src/main/java/sevensmurfs/rehub/controller/PasswordResetController.java

```diff
@@ -0,0 +1,56 @@
+package sevensmurfs.rehub.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import sevensmurfs.rehub.model.entity.PasswordReset;
+import sevensmurfs.rehub.model.message.request.PasswordResetRequest;
+import sevensmurfs.rehub.model.message.request.validator.PasswordResetValidator;
+import sevensmurfs.rehub.model.message.request.validator.UserRequestValidator;
+import sevensmurfs.rehub.service.PasswordResetService;
+
+@RestController
+@RequiredArgsConstructor
+@Slf4j
+public class PasswordResetController {
```

Comment:
No need for making separate controller when the endpoint is same as auth. You can move all code from class to Auth controller.